app/cli.js

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env node
 
-const config = require('./app/cliConfig');
-const db = require('./app/dbConnection');
-const wetParser = require('./app/wetParser.js');
-const algorithms = require('./app/algorithms');
+const config = require('./Cli/config.js');
+const db = require('./DbConnection/functions.js');
+const wetParser = require('./FileParser/functions.js');
+const algorithms = require('./Algorithms/functions.js');
```

Comment:
sound to me like it would make sense to rename the `functions.js` files to `index.js`. This would allow us to just import the directory.

Also I would go with lowercase directory names, but that is just a papercut.